pages/common/jest.md

```diff
@@ -0,0 +1,23 @@
+# jest
+
+> Delightful JavaScript Testing.
+
+- Run all tests:
+
+    jest
+
+- Run tests filtered by spec name:
+
+    jest -t name-of-spec
+
+- Run tests relating to uncommitted files:
```

Comment:
Is there an expanded version of the flag ? `-o` does not match with uncommitted.

Reply:
Yeah, it's `--onlyChanged`. Do we have a convention for specifying expanded flags alongside the short version?

Reply:
Yes, we do.  https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines pt. 4.

The heuristic is, keep the single word flag if it is clear enough or there is a word in the description starting with it. Otherwise, if there is a spelled out version, use that.